RELEASE.md

```diff
@@ -0,0 +1,5 @@
+Release type: bugfix
+
+- **bug** [merge_date_url] don't blow up when passed a URL setting that contains "-" (eg. "%-d").
```

Comment:
Regarding formatting, can we put colons after `**bug**, **support**`, etc, and capitalize the first word of each item? (in this case, `Don't blow up […]`)

Reply:
Fixed! I've updated the rest of the Changelog to match this format too.